src/rentals/rentals.resolver.ts

```
@@ -1,55 +1,47 @@
+        private readonly rentalService: RentalsService,
+        private userService: UsersService,
+        private bookService: BooksService,
+    ) {}
+
+    @Query(() => [Rental], { name: 'rentals' })
+    async getAllRentals(@Args('params') params: FindRentalInput): Promise<Rental[] | void> {
+        return await this.rentalService.getAllRentals(params);
+    }
+
+    @Query(() => Rental, { name: 'rental' })
+    async findRentalById(@Args('params') { _id }: FindRentalInput): Promise<Rental | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)